index.html

```diff
@@ -1433,6 +1433,10 @@ <h3>
           (e.g., amongst a list of other applications, or as a label for an
           icon).
         </p>
+        <p>
+          The [=accessible name=] of the web application is equal to the value
```

Comment:
Maybe we should phrase this as:

> The {{WebAppManifest/name}} member serves as the [=accessible name=] of an [=installed web application=]. 

@aarongustafson wdyt? ... same with accessible description.